ecms_base/themes/custom/ecms/templates/content/node--event--teaser.html.twig

```diff
@@ -91,12 +94,6 @@
 {% set end_time = node.field_event_date.end_value|date('g:ia') %}
 {% set datetime_endtime_format = node.field_event_date.end_value|date('H:i:s') %}
 
-{% if start_time != end_time %}
-  {% set time = start_time ~ ' - ' ~ end_time %}
-{% else %}
-  {% set time = start_time %}
-{% endif %}
```

Comment:
Is `time` not being used below this? I am looking for the removal or update of the variable. Maybe it wasn't being used at all.

Reply:
Yeah the variable was never passed to patternlab so I removed the block of code. I think I changed strategies when building it out and forgot to remove it.

Reply:
Got it